units/migrate_parts.py

```diff
@@ -0,0 +1,44 @@
+import json
+import sys
+
+
+
+def main():
+    master_array = []
+    
+    with open("../master_part_list_2.csv", "r") as parts_file:
+        lines = parts_file.readlines()
+        first_line = True
+        for line in lines:
+            if first_line:
+                first_line = False
+                continue
+            line = line.rstrip('\n')
+            master_array.append(parseLine(line))
+
+    with open("../master_part_list.json", "w") as json_file:
+        json.dump(master_array, json_file, indent=4)
```

Comment:
would be slightly faster and more memory efficient with the generator/comprehension methods I showed, but good enough - not going to block on that

Reply:
This is a one-off conversion script for a very specific file. This is the wrong hill to optimize on.